src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/WorkflowRunControllerTests.kt

```diff
@@ -57,6 +57,31 @@ class WorkflowRunControllerTests
         assertThat(sut.getWorkflowRunSummaries()).isEqualTo(workflowRunSummaries)
     }
 
+    @Test
+    fun `can get workflow run summary`()
+    {
+        val requestBody = """{"ref": "18f6c5267c08bf017b521a21493771c6d3e774a5", "reports": [{"name": "missing"}]}"""
+        val context = mock<ActionContext> {
+            on { getRequestBody() } doReturn requestBody
+        }
+
+        val mockAPIResponse = OrderlyServerResponse("mockAPIResponseText", 200)
+
+        val apiClient = mock<OrderlyServerAPI> {
+            on { post(any(), any<String>(), any()) } doReturn mockAPIResponse
+        }
+
+        val repo = mock<WorkflowRunRepository>()
+        val sut = WorkflowRunController(context, repo, apiClient, mock())
+        sut.getWorkflowRunSummary()
```

Comment:
Are you saying to remove the below code? Seems like we should keep that.
```
verify(apiClient).post(
            "/v1/workflow/summary/",
            requestBody,
            emptyMap()
        )
```

Reply:
We don't need to verify that post has been called on the api if instead we check the result of `sut.getWorkflowRunSummary()` - if that matches mockAPIResponse, we know the post has been called on the API (because that's how the mock is set up), and it additionally checks that the controller correctly passes the result out.

It would also mean changing the way the mock is set up to be more specific about its expectations, so that it would only return the mock response for the expected parameters: 
```
 val apiClient = mock<OrderlyServerAPI> {
            on { post(eq("/v1/workflow/summary/"), eq(requestBody), eq(emptyMap())) } doReturn mockAPIResponse
        }
``